.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: test CI
+
+on: [pull_request]
+
+jobs:
+  pytest_with_coverage:
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    strategy:
+      matrix:
+        python-version: [3.7]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install package dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pyyaml requests
+          for pathfile in $(find ./ -name "requirements*.txt" -type f); do
+             pip install -r $pathfile
+          done
```

Comment:
```suggestion
          python -m pip install -r app/requirements.txt
```
is enough I think.

Reply:
Line 21: The python lib configured by the actions/setup-python@v1 do have **pyyaml** and **requests** built-in. Maybe add these 2 packages in the requirement-dev.txt in the future?

Reply:
I see. So please keep `python -m pip install pyyaml requests`. pyyaml and requests should be add in the requiremnts.txt in the future.